charts/rancher-eks-operator/0.1.0/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can this be roleBinding only to cattle-system NS ? Or do we need access to other NS’s

Reply:
It needs to access cattle-global-data. That's where the secrets and EKSClusterConfigs are.

Reply:
Believe we need to make 2 role bindings then for both namespaces.